src/services/Users.php

```diff
@@ -801,6 +804,16 @@ public function suspendUser(User $user): bool
             return false;
         }
 
+        if ($moderatingUser) {
+            if (!$moderatingUser->can('moderateUsers')) {
+                throw new ForbiddenHttpException('User is not permitted to perform this action');
```

Comment:
Probably weird to be using `ForbiddenHttpException` in the service…but didn't seem like we had a non-http specific Forbidden exception.